### PR TITLE
fix(tutorials): Update API health check commands and add JSON parsing with jq

### LINKED ISSUE #548: feat(tutorials): Each command in a separate “code snippet”
## What

The [E2E adopter Journey tutorials](https://eclipse-tractusx.github.io/docs/tutorials/e2e/) provides plenty of commands which need to be executed. Currently, more than one commands are in a code snippet section.

```shell
command 1
command 2
```

This should be changed. One command per code snippet section.

```shell
command 1
```

```shell
command 2
```

## Why

This makes it more readable and also usable, because the copy paste functionality can be used

### LINKED ISSUE #560: fix(tutorials): delete one "empty" policy
## What

The [E2E adopter Journey tutorials](https://eclipse-tractusx.github.io/docs/tutorials/e2e/) provides some policies in the provide Data Section. In this section, two “empty” (with no constraints) policies are created. One of them should be deleted or adapted with some constraints.

- [First](https://eclipse-tractusx.github.io/docs/tutorials/e2e/boost/provideData#request-catalog---second-try)
- [Second](https://eclipse-tractusx.github.io/docs/tutorials/e2e/boost/provideData#create-first-contract-definition)

Even if the first one is an access and the second one a usage policy… I think it's just confusing

## Why

The current situation is more confusing than helping

## Note

Please keep in mind to check the whole workflow, if you chnage something

### LINKED ISSUE #562: fix(tutorials): port forwarding postgres is wrong
## What

There is a command for port forwarding in the [coding section](https://eclipse-tractusx.github.io/docs/tutorials/e2e/connect/deployComponents#inspect-the-databases). Right now it is

```shell
kubectl port-forward postgres-66677b8665-lxxp2  5432:5423
``` 

but it should be

```shell
kubectl port-forward postgres-66677b8665-lxxp2  5432:5432
```

## Why

The default port in the container should be 5432

### LINKED ISSUE #564: feat(tutorials): better formatting for the result of cURL commands
## What

There are plenty of cURL commands in the [tutorial](https://eclipse-tractusx.github.io/docs/tutorials/e2e/#skills-required). In some cases, it would be better to add. 

```shell
| jq
````
at the end of a command to get a better formatting in the result.

Please be aware if you want to provoke an error result this might not be helpful, but in the case of 200 OK, it just looks nicer

## Why

Nice to have

### PR DESCRIPTION
**Pull request by awellnitz-materna**
## Description
We were able to gather many impressions at the first TractusX Community Days. This also resulted in some suggestions for improvement.

With this PR we want to integrate the following suggestions.
- Executable commands should be in separate lines so that they can be easily copied on the website
- jq should be appended to the end of the whole reuqest as in MXD, this improves readability

Closes #564 
Closes #562 
Closes #548
Closes #560 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by stephanbcbauer — DISMISSED** [review]
LGTM

**Comment by stephanbcbauer** [comment]
@florianrusch-zf we would need a second reviewer ;) for the tutorials

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 1]
```diff
@@ -39,21 +39,38 @@ For the most bare-bones installation of the dataspace, execute the following com
 ```shell
 # get the tutorial including the config file for the cluster by cloning the repository locally
 git clone  https://github.com/eclipse-tractusx/tutorial-resources.git
+```
+```shell
 # will install the mxd directory under <current working directory>/tutorial-resources/mxd
 # the directory should contain the config file for kind: kind.config.yaml
 cd <path/of/mxd>
+```
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:47]:
```suggestion
```
```

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 2]
```diff
@@ -39,21 +39,38 @@ For the most bare-bones installation of the dataspace, execute the following com
...
+```shell
 # will install the mxd directory under <current working directory>/tutorial-resources/mxd
 # the directory should contain the config file for kind: kind.config.yaml
 cd <path/of/mxd>
+``` 
+
+```shell
 kind create cluster -n mxd --config kind.config.yaml
+```
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:51]:
```suggestion
```
```

**Review by florianrusch-zf — APPROVED** [review]

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 3]
```diff
@@ -39,21 +39,36 @@ For the most bare-bones installation of the dataspace, execute the following com
...
 # get the tutorial including the config file for the cluster by cloning the repository locally
 git clone  https://github.com/eclipse-tractusx/tutorial-resources.git
+```
+```shell
 # will install the mxd directory under <current working directory>/tutorial-resources/mxd
 # the directory should contain the config file for kind: kind.config.yaml
 cd <path/of/mxd>
+
+```shell
 kind create cluster -n mxd --config kind.config.yaml
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:49]:
````suggestion
kind create cluster -n mxd --config kind.config.yaml
```
````

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 4]
```diff
@@ -39,21 +39,36 @@ For the most bare-bones installation of the dataspace, execute the following com
 ```shell
 # get the tutorial including the config file for the cluster by cloning the repository locally
 git clone  https://github.com/eclipse-tractusx/tutorial-resources.git
+```
+```shell
 # will install the mxd directory under <current working directory>/tutorial-resources/mxd
 # the directory should contain the config file for kind: kind.config.yaml
 cd <path/of/mxd>
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:46]:
````suggestion
```shell
cd <path/of/mxd>
````

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
Sorry, I just wanted to remove the spaces and not the code fences 😅 the suggestion syntax is a little bit confusing regarding changing code fences

File: docs/tutorials/e2e/connect/deployComponents.md (hunk 3 above)
Comment (awellnitz-materna) [reply docs/tutorials/e2e/connect/deployComponents.md:49]: I have also just seen the pipeline :D

File: docs/tutorials/e2e/connect/deployComponents.md (hunk 4 above)
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:46]:
````suggestion
cd <path/of/mxd>
```
````

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 5]
```diff
@@ -39,21 +39,38 @@ For the most bare-bones installation of the dataspace, execute the following com
 ```shell
 # get the tutorial including the config file for the cluster by cloning the repository locally
 git clone  https://github.com/eclipse-tractusx/tutorial-resources.git
+```
+```shell
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:43]: Just to keep it consistent
````suggestion

```shell
````

File: docs/tutorials/e2e/connect/deployComponents.md [hunk 6]
```diff
@@ -148,8 +171,10 @@ assets, policies and contract definitions.
 In order to check that the connectors were deployed successfully, please execute the following commands in a shell:
 
 ```shell
-curl -X GET http://localhost/bob/health/api/check/liveness
-curl -X GET http://localhost/alice/health/api/check/liveness
+curl -X GET http://localhost/bob/health/api/check/liveness | jq
+```
+```shell
```
Comment (florianrusch-zf) [inline docs/tutorials/e2e/connect/deployComponents.md:176]:
````suggestion

```shell
````

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

**Review by awellnitz-materna — COMMENTED** [review]
Now it looks good again 😄

**Review by florianrusch-zf — APPROVED** [review]

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Pull request merged**